pom.xml

```diff
@@ -555,5 +555,12 @@
             <artifactId>selenium-chrome-driver</artifactId>
             <version>3.141.59</version>
         </dependency>
+
+        <!-- web3 -->
+        <dependency>
+            <groupId>org.web3j</groupId>
+            <artifactId>core</artifactId>
+            <version>4.8.4</version>
+        </dependency>
```

Comment:
Is this Maven dependency required for the `web3.min.js` to work? Or was it included because it will be used later in backend code (Java)?

Reply:
It is required for the backend code!